openmeter/app/kafka.go

```diff
@@ -0,0 +1,44 @@
+package app
+
+import (
+	"fmt"
+	"log/slog"
+
+	"github.com/confluentinc/confluent-kafka-go/v2/kafka"
+	"go.opentelemetry.io/otel/metric"
+
+	"github.com/openmeterio/openmeter/config"
+	pkgkafka "github.com/openmeterio/openmeter/pkg/kafka"
+	kafkametrics "github.com/openmeterio/openmeter/pkg/kafka/metrics"
+)
+
+// TODO: use ingest config directly?
+// TODO: use kafka config directly?
```

Comment:
Keep it as is for now. I'll refactor the Kafka config init/usage in services in order to adopt the `Producer/ConsumerConfig` sturct available in `pkg/kafka` pkg: https://github.com/openmeterio/openmeter/blob/main/pkg/kafka/config.go#L328